rollup.config.js

```diff
@@ -18,15 +18,14 @@ import livereload from "rollup-plugin-livereload";
 import { terser } from "rollup-plugin-terser";
 
 import packageJson from "./package.json";
+import { debug } from "svelte/internal";
 
 dotenv.config();
 const production = !process.env.ROLLUP_WATCH;
-const build = execSync("git rev-parse --short HEAD")
-  .toString()
-  .trim();
+const build = retrieveGitHead();
```

Comment:
Having the commit hash in the production build is quite useful. I'd change the logic a bit so that:

- `retrieveGitHead` returns the hash or `null` if the `execSync` fails
- `const build = retrieveGitHead() || "unknown"`